### PR TITLE
Migrate the repo to describe GH Actions CI setup

### PR DESCRIPTION
**Pull request by anatoliykmetyuk**
The next step would be to rename the repo to `dotty-ci`.

Why do we have `scala-docker/Dockerfile` in the repo? I haven't found any usages of it, so I deleted it as well.

### CODE REVIEW

File: dotty-docker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:19.10
```
Comment (smarter) [inline dotty-docker/Dockerfile:1]: 20.04 is out and is a long term support release, would be good to use that if possible.

File: dotty-docker/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
 # The default locale is "POSIX" which is just ASCII.
 ENV LANG C.UTF-8
```
Comment (smarter) [inline dotty-docker/Dockerfile:4]: the sbt version below is 1.3.6, might as well bump it to 1.3.10 to get the latest launcher

**Review by smarter — APPROVED** [review]
> Why do we have scala-docker/Dockerfile in the repo?

It was used to compile the backend a long time ago when it lived in a fork of scala/scala

**Comment by anatoliykmetyuk** [comment]
I remember I had troubles with the software on some different Ubuntu version, and version bumps require extra testing. FTTB I'll merge this to have something up-to-date. I've created an issue to bump the other stuff: #17.

**Pull request merged**